src/plugins/console/public/application/containers/editor/editor.tsx

```diff
@@ -11,7 +11,7 @@ import { debounce } from 'lodash';
 import { EuiProgress } from '@elastic/eui';
 
 import { EditorContentSpinner } from '../../components';
-import { Panel, PanelsContainer } from '../../../../../kibana_react/public';
+import { Panel, PanelsContainer } from '../../containers/split_panel';
```

Comment:
Please, can you add the export of `Panel` and `PanelsContainer` from the `containers/index.ts` and change the corresponding imports. Thanks)